docs/api/client/auth_handler.rst

```diff
@@ -1,5 +1,5 @@
 :mod:`letsencrypt.client.auth_handler`
---------------------------------
+--------------------------------------
```

Comment:
Thanks for taking care of this... I didn't actually know this rule... and haven't spent that much time on the documentation.  I won't make this mistake again.